spec/rails_helper.rb

```diff
@@ -50,7 +50,17 @@
   #     RSpec.describe UsersController, :type => :controller do
   #       # ...
   #     end
-  #
+  def login(user)
+    client = create(user.to_s)
```

Comment:
Since the variable 'client' isn't being called within the login method, you can delete line 54.